docs/core/tools/csproj.md

```diff
@@ -67,8 +67,17 @@ Setting this property to `false` will override implicit inclusion and the behavi
 This change does not modify the main mechanics of other includes. However, if you wish to specify, for example, some files to get published with your app, you can still use the known mechanisms in *csproj* for that (for example, the `<Content>` element).
 
 ### Recommendation
-With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (runtime, NuGet packaging, etc.)
+With csproj, we recommend that you remove the default globs from your project and only add file paths with globs for those artifacts that your app/library needs for various scenarios (for example, runtime and NuGet packaging).
 
+## How to see the whole project as MSBuild sees it
```

Comment:
One major wrinkle that may be worth mentioning here (it keeps biting me and I should know better than anyone):

If your project has multiple `TargetFramework`**`s`**, just running `dotnet msbuild /pp` on it will preprocess the "outer" build, which isn't very interesting--it just dispatches to the inner builds. You usually want to also specify a TFM, for example with `dotnet msbuild /p:TargetFramework=netcoreapp2.0 /pp`. Note that the convenience `dotnet -f {tfm}` doesn't work, you have to specify the framework as an MSBuild property.